tests/installation/reconnect_s390.pm

```diff
@@ -40,14 +40,14 @@ sub handle_login_not_found {
 
 sub run {
     my $login_ready = check_var('VERSION', 'Tumbleweed') ? qr/Welcome to openSUSE Tumbleweed 20.*/ : qr/Welcome to SUSE Linux Enterprise .*\(s390x\)/;
+    console('installation')->disable_vnc_stalls;
```

Comment:
this is already done before in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L181

Reply:
maybe. But pulling it out of the "if-backend" will make code cleaner and might even help ;-) I think I need to conduct some more experiments for statistics.